aries-site/src/pages/whats-new.mdx

```diff
@@ -52,14 +52,87 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 4 - January 21, 2022
+
+### Roadmap Updates
+- **Tag** - Completed guidance for [Tag](https://design-system.hpe.design/components/tag).
+- **Navigation** 
+   - Completed research for [Ascending Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2382%3A13827).
+   - Completed ideations for [Ascending Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2725%3A18556).
+- **Button** - [Updated font color on default button active states](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component) from default text to text-strong.
+- **Page Layouts** 
+   - Completed Dashboard template for a [2 column layout](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2231%3A17992).
+   - Shared status update with the Design System Council on January 20, 2022.
+   - Completed documentation around the definition, scope and anatomy of Page Layouts.
+   - Created guidelines for the Page Container, including defining 3 types of content widths: Wide, Narrow and Full.
+   - Defined new breakpoints and page margins for each breakpoint.
+   - Work in progress for Page Layout templates, starting with 2 column, 3 column and multi-column Dashboards.
+
+### Stay current with the latest changes
+- **Voice and Tone** - Updates in progress for Voice & Tone page.
+- **Accessibility** - Updates in progress for Accessibility page to provide more resources and guidance.
+- **Persistent Sidebar to be deprecated by end of year** - In preparation for the deprecation of the Persistent Sidebar in Design System, we have removed any examples and documentation surrounding its use.
+
+### Latest Releases
+- **Grommet v2.20.0** - Release includes the NEW Tag component along with enhancements to Calendar, Carousel, DateInput, Form, Video; fixes applied to Anchor, Button, Calendar, Clock, Select, Text, and TextInput. Full details about this release are available in the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.20.0).
+
+## December 24 - January 3, 2022
+
+Happy Holidays 🎉
+
+## December 20 - 23, 2021
+
+### Roadmap Updates
+- **Page Layouts** - Commenced explorations on [Dashboards within Page Layouts](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2159%3A17698).
+- **NameValueList** - Created variant of [editable horizontal form fields with NameValue lists](https://design-system.hpe.design/components/namevaluelist?q=name#name-value-pair-as-a-read-only-display-for-text-inputs).
```

Comment:
Correct link, keep in same window since linking within DS site.

```suggestion
- **NameValueList** - Created variant demonstrating how [editable horizontal form fields](/components/namevaluelist#name-value-pair-as-a-read-only-display-in-horizontal) may be displayed within Name-Value Lists.
```